sympy/stats/tests/test_joint_rv.py

```diff
@@ -236,10 +236,20 @@ def test_expectation():
     m = Normal('A', [x, y], [[1, 0], [0, 1]])
     assert simplify(E(m[1])) == y
 
-@XFAIL
+
 def test_joint_vector_expectation():
     m = Normal('A', [x, y], [[1, 0], [0, 1]])
-    assert E(m) == (x, y)
+    assert E(m).simplify() == Matrix([[x], [y]])
+
+    # issue 19552
+    s1, s2, s3, s4, s5, s6 = symbols('sigma1:7', positive=True)
```

Comment:
This code should work without `positive=True` as well. On my computer, it hangs.

Reply:
May be it can be an issue with the Integrals?

Reply:
I called checked it on one of integral and it produces two diff results based in assumptions:
```py
>>> sigma1, sigma2, sigma3 = symbols('sigma1:4')
>>> N = Normal("N", [x, y, z], diagonalize_vector(Matrix([s1, s2, s3])))
>>> expr = sqrt(2)*exp((-z + N[2])*(z/2 - N[2]/2)/sigma3 + (-y + N[1])*(y/2 - N[1]/2)/sigma2 + (-x + N[0])*(x/2 - N[0]/2)/sigma1)*N[0]/(4*pi**(3/2)*sqrt(sigma1)*sqrt(sigma2)*sqrt(sigma3))
>>> ans1 = integrate(ex, (N[0], -oo, oo), (N[1], -oo, oo), (N[2], -oo, oo))
>>> ans1
oo*sign(exp(-z**2/(2*sigma3) + z*N[2]/sigma3 - N[2]**2/(2*sigma3) - y**2/(2*sigma2) + y*N[1]/sigma2 - N[1]**2/(2*sigma2) - x**2/(2*sigma1) + x*N[0]/sigma1 - N[0]**2/(2*sigma1))*N[0]/(sqrt(sigma1)*sqrt(sigma2)*sqrt(sigma3)))
>>> ans1.simplify() # this hangs on my PC
```

Changing the assumption:
```py
>>> sigma1, sigma2, sigma3 = symbols('sigma1:4', positive=True)
>>> N = Normal("N", [x, y, z], diagonalize_vector(Matrix([s1, s2, s3])))
>>> expr = sqrt(2)*exp((-z + N[2])*(z/2 - N[2]/2)/sigma3 + (-y + N[1])*(y/2 - N[1]/2)/sigma2 + (-x + N[0])*(x/2 - N[0]/2)/sigma1)*N[0]/(4*pi**(3/2)*sqrt(sigma1)*sqrt(sigma2)*sqrt(sigma3))
>>> ans2 = integrate(ex, (N[0], -oo, oo), (N[1], -oo, oo), (N[2], -oo, oo))
>>> ans2
x*(2 - erfc(sqrt(2)*z/(2*sqrt(sigma3))))/2 + x*erfc(sqrt(2)*z/(2*sqrt(sigma3)))/2
>>> ans2.simplify()
x
```

Reply:
It's not probably an issue, may be that's how it is. It depends on the default assumptions of symbols.
```python
IPython console for SymPy 1.7.dev (Python 3.6.9-64-bit) (ground types: python)

These commands were executed:
>>> from __future__ import division
>>> from sympy import *
>>> x, y, z, t = symbols('x y z t')
>>> k, m, n = symbols('k m n', integer=True)
>>> f, g, h = symbols('f g h', cls=Function)
>>> init_printing()

Documentation can be found at https://docs.sympy.org/dev


In [1]: from sympy.stats import *                                               

In [2]: x.is_complex                                                            

In [3]: x.is_positive                                                           

In [4]: x.is_real                                                               

In [5]: x.is_negative                                                           

In [6]: k.is_positive                                                           

In [7]: k.is_integer                                                            
Out[7]: True
```
Results with correct assumptions should be correct in any case, IMHO.

Reply:
See, https://github.com/sympy/sympy/pull/19957/files#diff-f5bd154f958178a67c12702770445325R36-R45 
Type checks can create problems, not for this PR but still a design issue in my opinion.

Reply:
> May be it can be an issue with the Integrals?

Did you check the integral produced by the univariate RV? I works perfectly. Is the integral different in the multivariate case? This kind of multivariate RV is simply a tuple of three independent RVs, I'm a bit surprised by the different behaviour of `Integral`.

Reply:
Commenting out the lines from https://github.com/sympy/sympy/pull/19957#discussion_r470545391
I get the following results:
```py
>>> x, s1 = symbols('x, sigma')
>>> N = Normal('N', x, s1)
>>> E(N)
Piecewise((-x*(-2*sqrt(2)*sigma1*exp(-x**2/(2*sigma1**2))/x - 2*sqrt(pi)*(2 - erfc(sqrt(2)*x/(2*sigma1))))/(4*sqrt(pi)) - x*(2*sqrt(2)*sigma1*exp(-x**2/(2*sigma1**2))/x - 2*sqrt(pi)*erfc(sqrt(2)*x/(2*sigma1)))/(4*sqrt(pi)), ((2*Abs(arg(sigma1)) < pi/2) | ((2*Abs(arg(sigma1)) <= pi/2) & (Abs(4*arg(sigma1) - 2*arg(x)) < pi)) | ((2*Abs(arg(sigma1)) < pi/2) & (Abs(4*arg(sigma1) - 2*arg(x)) <= pi)) | ((2*Abs(arg(sigma1)) < pi/2) & (Abs(4*arg(sigma1) - 2*arg(x)) < pi))) & ((2*Abs(arg(sigma1)) < pi/2) | ((2*Abs(arg(sigma1)) <= pi/2) & (Abs(-4*arg(sigma1) + 2*arg(x) + 2*pi) < pi)) | ((2*Abs(arg(sigma1)) < pi/2) & (Abs(-4*arg(sigma1) + 2*arg(x) + 2*pi) <= pi)) | ((2*Abs(arg(sigma1)) < pi/2) & (Abs(-4*arg(sigma1) + 2*arg(x) + 2*pi) < pi)))), (Integral(sqrt(2)*N*exp(-(N - x)**2/(2*sigma1**2))/(2*sqrt(pi)*sigma1), (N, -oo, oo)), True))
```
The results differ because of the ad-hoc methods that are present in `Normal`.

Reply:
OK. All right, this conversation is resolved for me.